packages/babel-traverse/src/scope/index.js

```diff
@@ -830,7 +832,7 @@ export default class Scope {
         return scope;
       }
     } while ((scope = scope.parent));
-    throw new Error("We couldn't find a Function or Program...");
+    return null;
```

Comment:
Scope's `getFunctionParent` throws. paths' `getFunctionParent` doesn't throw and silently fails by returning `null`. Looks like you have swapped this behaviour.